### PR TITLE
[AWS] Add reference to API requests and pricing info

### PR DESCRIPTION
**Pull request by alaudazzi**
This PR:
- updates the [API requests](https://www.elastic.co/docs/current/integrations/aws#api-requests) page  by moving these two lines to the top of the table:
`Each of these APIs may generate extra charges on your AWS Account. Please refer to [AWS Pricing](https://aws.amazon.com/pricing) for more information.`
- adds a reference to the [API requests](https://www.elastic.co/docs/current/integrations/aws#api-requests) in the **IMPORTANT** note of each page listed [here](https://github.com/elastic/integrations/issues/11171#issue-2533472542)

Closes [#11171](https://github.com/elastic/integrations/issues/11171)

### CODE REVIEW

File: packages/aws/manifest.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-format_version: 3.0.0
+format_version: 3.0.1
 name: aws
 title: AWS
 version: 2.27.0
```
Comment (ishleenk17) [inline packages/aws/manifest.yml:4]:
```suggestion
version: 2.28.0
```

File: packages/aws_bedrock/manifest.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-format_version: "3.0.2"
+format_version: "3.0.3"
 name: aws_bedrock
 title: Amazon Bedrock
 description: Collect Amazon Bedrock model invocation logs and runtime metrics with Elastic Agent.
```
Comment (ishleenk17) [inline packages/aws_bedrock/manifest.yml:4]: You should not change the format version. Keep it as 3.0.2 and change the version to 0.11.1 as I suggested above as well

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Review by ishleenk17 — APPROVED** [review]

**Review by ShourieG — APPROVED** [review]
CODEOWNER review.

**Pull request merged**